introduction.md

```diff
@@ -1,34 +1,79 @@
-# Week 01 - Introduction 
+# Week 01 - Introduction
 
-### Goals 
-- Gain insights about the kids, 
+### Goals
+- Gain insights about the kids,
 - Establish context for the program,
 - Inspire them, to look forward for the next session
 
-### Survey 
+### Survey
 - Should be fun to engage in
 - Gain insights on prior knowledge, interests, expereinces, behaviour
 #### Questions
 - What comes to your mind when we say "computers" or "technology"?
 - How many of you have used a computer or phone or calculater in the past?
-- Have you ever wondered how they work or thought about building something like them or doing something with them? 
-- What do you want to become in your life? 
+- Have you ever wondered how they work or thought about building something like them or doing something with them?
+- What do you want to become in your life?
 
-- We should check if our upcoming expames and analogies are relavent to the kids or not. 
+- We should check if our upcoming expames and analogies are relavent to the kids or not.
 
-### Morning Routine 
-Description : An unplugged exercise to show the kids what an algorithm or generic code looks like, by breaking down their morning 
-routine of getting up from bed and going to school. We can introduce code structure, flow, if statements, loops, through this exercise 
+Description : An unplugged exercise to show the kids what an algorithm or generic code looks like
 
-### History of CS 
-Description : Walk the kids through the history of computers, to answer questions like "why were they invented", 
-"what problems do they solve", "why do we need them", "how did they evolve over time". The goal here is to establish the context of why 
-computers exist in the first place, but in a very story-like fashion. 
+Goal: A metaphor that we can map back to coding.
+We can introduce code structure, flow, if statements, loops, through this exercise and keep getting back to this example throughout the course.
+
+For week1, we can stop with using this to introduce the concept of `algorithm`, and maybe
+`code` and `programming language`.
+
+*Example#1*
+- Breaking down their morning routine of getting up from bed and going to school
+What we expect more or less - get up => brush => shower =>
+breakfast => wait for the right bus (loops/conditionals) => get to class =>
+greet your friends (function concept), etc
+
+*Example#2*
+- Planning a birthday party for a friend. <TODO: @Madhu94 add a short description>
+
+*Example#3* (From code.org)
+- Planting a seed and growing a tree
+
+We will encourage students to contribute answers and build from that.
+
+(We need a wide range of examples here, so that we can alter our examples based on the
```

Comment:
Again, wide range of examples for the Introduction itself or for the entire program? If the later, we might as well create a septate new document where we can put all of these examples together, and use one per session. Also, for these particular examples, I don't think we really have to depend on the survey to pick them. They are too common in everyone's life, so I don't think we would have to worry about any kid not relating to any of these examples (morning routine, birthday, planting a tree etc) because we are coming up with them by thinking about what do all of the kids do or go through

Reply:
I think the line was misplaced and I meant this for the analogies section below (with the films, moon landing, etc). I will fix it